src/components/DocumentationTopic/DocumentationNav/HierarchyCollapsedItems.vue

```diff
@@ -113,11 +114,13 @@ $dropdown-transition-duration: 250ms;
 $hierarchy-dropdown-border-width: 0px !default;
 $hierarchy-dropdown-box-shadow: 0 1px 4px -1px var(--color-figure-gray-secondary) !default;
 
-$tail-width: 1rem;
+$tail-width: 1rem * 0.5;
```

Comment:
Because `$tailwidth` is never used at it's base value, its only used to be divided by 2. So instead of shoving a bunch of `math.div(` calls, I just simplified it.